scripts/pre_commit_linter.py

```diff
@@ -488,8 +726,11 @@ def _lint_css_files(
     stylelint_cmd_args = [
         node_path, stylelint_path, '--config=' + config_path]
     result_list = []
+    if verbose_mode_enabled is False:
```

Comment:
`if not verbose_mode_enabled`

Ditto below

Reply:
Done